cfme/intelligence/chargeback/rates.py

```diff
@@ -224,19 +207,65 @@ def delete(self, cancel=False):
         """
         view = navigate_to(self, 'Details', wait_for_view=True)
         view.toolbar.configuration.item_select('Remove from the VMDB', handle_alert=(not cancel))
+        view = self.create_view(navigator.get_class(self.parent, 'All').VIEW)
+        assert view.is_displayed
+        view.flash.assert_no_error()
+
+
+@attr.s
+class ComputeRateCollection(BaseCollection):
+    ENTITY = ComputeRate
+    RATE_TYPE = 'Compute'
```

Comment:
I'm not a fan of defining this both in the Collection and Entity classes, though it may be useful to access from either without having to determine if you need to call `thing.RATE_TYPE` or `thing.parent.RATE_TYPE` when you have an instance instead of the collection.

I'm thinking a decent approach would be to define it here as `RATE_TYPE = ENTITY.RATE_TYPE`

Reply:
Good catch. Updated